app/src/main/java/com/twilio/video/app/ui/room/RoomViewModel.kt

```diff
@@ -108,6 +114,14 @@ class RoomViewModel(
         }
     }
 
+    private fun checkLocalMedia() {
```

Comment:
Can we rename this method to checkLocalMedia and the RoomViewEvent to CheckPermissions?